### PR TITLE
PHPStan - Level 6

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: src/Client/Response/Base/Response.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ class Response
     /**
      * Array detailing which parameters were invalid.
      *
-     * @var array[]
+     * @var array<int, string>
```
Comment (OskarStark) [inline src/Client/Response/Base/Response.php:55]: unsure about this array structure @slunak

**Comment by OskarStark** [comment]
> Method Serhiy\Pushover\Client\Response\RetrieveGroupResponse::setUsers() has parameter $users with no value type specified in iterable type array.

Can you tell me how the array looks like passed to this method?

File: src/Client/Response/Base/Response.php (hunk 1 above)
Comment (slunak) [reply src/Client/Response/Base/Response.php:55]: This is just an array of strings.

Example:

`{"user":"invalid","errors":["user identifier is invalid"], "status":0,"request":"5042853c-402d-4a18-abcb-168734a801de"}`

https://pushover.net/api#response

**Comment by slunak** [comment]
> > Method Serhiy\Pushover\Client\Response\RetrieveGroupResponse::setUsers() has parameter $users with no value type specified in iterable type array.
> 
> Can you tell me how the array looks like passed to this method?

https://pushover.net/api/groups#show

```json
{
  "status": 1,
  "request": "4044780a-8b6d-4416-9c0f-f123bc1b6450",
  "name": "My Delivery Group",
  "users": [
    {
      "user": "uoJCttEFQo8uoZ6REZHMGBjX2pmcdJ",
      "device": null,
      "memo": "Jim",
      "disabled": false
    },
    {
      "user": "uZgLtMeFSRgdxLbpWbDDPM9v4wmEKu",
      "device": "iphone",
      "memo": "Mary",
      "disabled": false
    }
  ]
}
```
It is users part of the above json. It is `json_decode`ed, so array contains arrays of users.

File: src/Client/Response/Base/Response.php (hunk 1 above)
Comment (OskarStark) [reply src/Client/Response/Base/Response.php:55]: Then it is correct 👍🏻

Comment (slunak) [reply src/Client/Response/Base/Response.php:55]: `array<string>` would also work, right?

**Comment by OskarStark** [comment]
Yes array shapes 👌🏻

File: src/Client/Response/Base/Response.php (hunk 1 above)
Comment (OskarStark) [reply src/Client/Response/Base/Response.php:55]: Or string[]

**Review by slunak — APPROVED** [review]

**Pull request merged**